tests/testthat/test-stat_likelihoods.R

```diff
@@ -0,0 +1,116 @@
+set.seed(1231)
+chains <- c(1, 1, 4, 7)
+test_that("Analytical chain size distributions are numerically correct", {
```

Comment:
missing test for `pois_size_ll`?

Reply:
Nice catch. Fixed in 0b2005a2a7266bae04e1649d17c818c22857ea48. Because there's a mix of outputs from `likelihood()` based on inputs, I've added extra checks to test for object classes returned.